content/en/_index.md

```diff
@@ -1138,33 +1138,31 @@ end
 {{< /content-tab >}}
 
 {{< content-tab header="Terraform" >}}
-#### Don't use `null_resource`
+
+### Don't use `null_resource`
 
 Do not include CLI commands (such as gcloud or kubectl) inside
 of your sample via the [`null_resource`](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource) resource.
 
 There are some products without Terraform support. Do not include
 those products in Terraform samples.
 
-#### No global variables
+### No global variables
 
 Don't include global Terraform variables (`var.VARIABLE_NAME`).
 
-{{% alert title="Explanation" %}}
 Global variables become part of the API exposed via Terraform.
 Therefore, global variables help enable users to use Terraform samples
 not as snippets but as direct module references. 
-By disallowing global variables, we disable this avenue of customization.
-This reduces the risk of users developing code that is
-accidentally dependent on our samples.
+By not using global variables in the samples in our cloud.google.com docs, 
+we reduce the risk of users developing code that depends directly on the samples.
 
-Instead of including global variables, hard code the resource names and
+Instead of global variables, hard code the resource names and
 parameter values.
 
 Local variables are allowed in Terraform samples
 because local variables are an internal implementation and therefore aren't
 part of the API exposed via Terraform. See [local variables](#locals).
-{{% /alert %}}
 
 #### Local variables {#locals}
```

Comment:
Shouldn't **Local variables** and **Resource references** also have their headings fixed?

Reply:
I updated so that they are consistent. It's not easy to see all the headers to be honest.